03_LabJat/letterFreequencyCount.go

```diff
@@ -0,0 +1,31 @@
+package main
+
+import (
+	"fmt"
+	"io"
+	"os"
+	"sort"
+	"strconv"
+	"strings"
+)
+
+var out io.Writer = os.Stdout
+
+func letters(input string) map[rune]int {
+	m := map[rune]int{}
+	for _, c := range input {
+		m[c]++
+	}
+	return m
+}
+func sortedLetters(m map[rune]int) []string {
```

Comment:
nit: put a blank line between the end of one function and the start of the next. Do this across the whole file.